README.md

```diff
@@ -16,11 +16,29 @@ Irmin is an OCaml library for building mergeable, branchable distributed data st
 ### Documentation
 Documentation can be found online at [https://mirage.github.io/irmin](https://mirage.github.io/irmin)
 
+#### Prerequisites Note 
+  Please ensure to install the latest Opam version to have the latest packages. (As of now,opam's version can be equal to or higher versions than '2.0.4').
+  
+  Check the Ocaml version greater or equal to '4.09.0' with the help of command 
+        
+    opam list   
+
+  If the OCaml version is not the specified version, then use the below commands.
+
+    opam switch create version (eg. version could be 4.09.0)
+    eval $(opam env)
+  
+  We hope, with this prior information you can install the irmin packages successfully.
```

Comment:
I don't think this sentence is necessary :)

Reply:
ok. This can be avoided. If we can show how  to install  opam of specified vesrion.

Reply:
How to install opam belongs to the opam documentation and installation instructions. If you want to make a prerequisite section here, the point is to mention:
- Which minimum version of opam is needed.
- Which minimum version of OCaml is needed.
Then redirect the user to the corresponding documentations. We should not duplicate documentation that is available elsewhere.